benchmark/sql_regressions/run_regressions.sh

```diff
@@ -0,0 +1,211 @@
+      dolt checkout nightly
+      table_prefix="nightly"
+      elif [ "$JOB_TYPE" == "release" ]; then
+        dolt checkout releases
+        table_prefix="releases"
+      else fail Unknown JOB_TYPE specified;
+    fi
+
+    previously_tested_version=$(dolt sql -r csv -q "select * from ${table_prefix}_dolt_results where version = '$DOLT_VERSION' limit 1;"| wc -l | tr -d '[:space:]')
+
+    if [ "$previously_tested_version" != 1 ]; then
+      echo "Results for dolt version $DOLT_VERSION already exist in Liquidata/dolt-sql-performance, $previously_tested_version != 1" && \
+      echo $result_query_output && \
+      exit 1;
+    fi
+
+    echo $previously_tested_version
+    dolt checkout master
+}
+
+function import_one_nightly() {
+    test_num="$1"
+    dolt table import -u nightly_dolt_results ../"$logictest_main"/temp/parsed"$test_num".json
+    dolt add nightly_dolt_results
+    dolt commit -m "update dolt sql performance results ($DOLT_VERSION) ($test_num)"
+}
+
+function import_nightly() {
+    dolt checkout nightly
+    seq 1 $TEST_N_TIMES | while read test_num; do
+        import_one_nightly "$test_num"
+    done
+    dolt sql -r csv -q "\
+select version, test_file, line_num, avg(duration) as mean_duration, result from dolt_history_nightly_dolt_results where version=\"${DOLT_VERSION}\" group by line_num;\
+" > nightly_mean.csv
+    dolt table import -u nightly_dolt_mean_results nightly_mean.csv
+    dolt add nightly_dolt_mean_results
+    dolt commit -m "update dolt sql performance mean results ($DOLT_VERSION)"
+    dolt push origin nightly
+
+    dolt checkout regressions
+    dolt merge nightly
+    dolt add .
+    dolt commit -m "merge nightly"
+    dolt push origin regressions
+
+    dolt checkout releases
+    dolt sql -r csv -q "\
+select * from releases_dolt_mean_results;\
+" > releases_mean.csv
+    rm -f regressions_db
+    touch regressions_db
+    sqlite3 regressions_db < ../"$logictest"/regressions.sql
+    cp ../"$logictest"/import.sql .
+    sqlite3 regressions_db < import.sql
+    echo "Checking for test regressions..."
+
+    duration_query_output=`sqlite3 regressions_db 'select * from releases_nightly_duration_change'`
+    result_query_output=`sqlite3 regressions_db 'select * from releases_nightly_result_change'`
+
+    duration_regressions=`echo $duration_query_output | sed '/^\s*$/d' | wc -l | tr -d '[:space:]'`
+    result_regressions=`echo $result_query_output | sed '/^\s*$/d' | wc -l | tr -d '[:space:]'`
+
+    if [ "$duration_regressions" != 0 ]; then echo "Duration regression found, $duration_regressions != 0" && echo $duration_query_output && exit 1; else echo "No duration regressions found"; fi
+    if [ "$result_regressions" != 0 ]; then echo "Result regression found, $result_regressions != 0" && echo $result_query_output && exit 1; else echo "No result regressions found"; fi
+}
+
+function build_dolt_release() {
+    pushd ../../ && \
+    rm ./.ci_bin/dolt && \
```

Comment:
Maybe but this into `./ci_bin/dolt_release` or something, call this function `with_dolt_release` do it like:

```sh
function build_dolt_release() {
    (
        cd ../../
        if ! [ -x ./.ci_bin/dolt_release/dolt ]; then
            curl -A CURL_USER_AGENT:-dolt-installer -fL "$DOLT_RELEASE_URL" > dolt.tar.gz
            tar zxf dolt.tar.gz
            install dolt-linux-amd64/bin/dolt ./.ci_bin/dolt_release/
        fi
    )
    echo "Finished installing dolt from release:"
    export PATH=`pwd`"/ci_bin/dolt_release"
    dolt version
}
```